### PR TITLE
Show Ticket Link in the My Event List

### PR DESCRIPTION
**Pull request by rafsuntaskin**
Add Tickets link in Events Title
[ET-792]

[ET-792]: https://moderntribe.atlassian.net/browse/ET-792

### CODE REVIEW

File: src/views/shortcodes/my-attendance-list.php [hunk 1]
```diff
@@ -22,6 +22,9 @@
 					<span class="datetime">(<?php echo $start_date; ?>)</span>
 				<?php endif; ?>
 			</a>
+			<a href="<?php echo esc_url( get_permalink( $id ). 'tickets' ); ?>" target="_blank">
```
Comment (tr1b0t) [inline src/views/shortcodes/my-attendance-list.php]: :no_entry_sign: Concat operator must be surrounded by a single space

<sub>hash: 75272948850aa8b4a1a925eadc2ba8d2</sub>

File: src/views/shortcodes/my-attendance-list.php [hunk 2]
```diff
@@ -22,6 +22,9 @@
...
+                		- <span><?php esc_html_e( 'View Tickets', 'event-tickets' ) ?></span>
```
Comment (tr1b0t) [inline src/views/shortcodes/my-attendance-list.php]: * :no_entry_sign: Tabs must be used to indent lines; spaces are not allowed
* :no_entry_sign: Inline PHP statement must end with a semicolon

<sub>hash: 8e3bf4c6ad275ab7c2e009543425fef8</sub>

File: src/views/shortcodes/my-attendance-list.php [hunk 3]
```diff
@@ -22,6 +22,9 @@
...
+            		</a>
```
Comment (tr1b0t) [inline src/views/shortcodes/my-attendance-list.php]: :no_entry_sign: Tabs must be used to indent lines; spaces are not allowed

<sub>hash: ce7e6169731d6120c9fc57e6f21197ad</sub>

File: src/views/shortcodes/my-attendance-list.php (hunk 1 above)
Comment (sc0ttkclark) [reply src/views/shortcodes/my-attendance-list.php:25]:
```suggestion
			<a href="<?php echo esc_url( get_permalink( $id ) . 'tickets/' ); ?>" target="_blank">
```

File: src/views/shortcodes/my-attendance-list.php (hunk 2 above)
Comment (sc0ttkclark) [reply src/views/shortcodes/my-attendance-list.php:26]:
```suggestion
                		- <span><?php esc_html_e( 'View Tickets', 'event-tickets' ); ?></span>
```

File: src/views/shortcodes/my-attendance-list.php [hunk 4]
```diff
@@ -22,6 +22,9 @@
 					<span class="datetime">(<?php echo $start_date; ?>)</span>
 				<?php endif; ?>
 			</a>
+			<a href="<?php echo esc_url( get_permalink( $id ) . 'tickets/' ); ?>" target="_blank">
+                		- <span><?php esc_html_e( 'View Tickets', 'event-tickets' ); ?></span>
+            		</a>
```
Comment (sc0ttkclark) [inline src/views/shortcodes/my-attendance-list.php:27]:
```suggestion
			<a href="<?php echo esc_url( get_permalink( $id ) . 'tickets/' ); ?>" target="_blank">
				- <span><?php esc_html_e( 'View Tickets', 'event-tickets' ); ?></span>
			</a>
```

**Pull request closed without merging**